content/tutorials/spatial_standards_vector/index.md

```diff
@@ -293,16 +293,16 @@ Let’s just look at the top 7 lines of the file:
     "type": "FeatureCollection",
     "name": "habitatstreams",
     "features": [
-    { "type": "Feature", "properties": { "river_name": "WOLFPUTBEEK", "source": "VMM" }, "geometry": { "type": "LineString", "coordinates": [ [ 4.0532635, 50.8196905 ], [ 4.0532327, 50.8197202 ], [ 4.0530778, 50.8197594 ], [ 4.0528708, 50.8199422 ], [ 4.052834, 50.8201498 ], [ 4.0528767, 50.8204559 ] ] } },
-    { "type": "Feature", "properties": { "river_name": "OUDE KALE", "source": "VMM" }, "geometry": { "type": "LineString", "coordinates": [ [ 3.5931564, 51.0803318 ], [ 3.5930966, 51.0803266 ], [ 3.5927771, 51.0802782 ], [ 3.5926209, 51.080259 ], [ 3.5925707, 51.0802465 ], [ 3.5925106, 51.0802316 ], [ 3.592303, 51.0801396 ], [ 3.5921047, 51.0800302 ], [ 3.5920091, 51.0799694 ], [ 3.5919755, 51.0799432 ], [ 3.5919328, 51.07991 ], [ 3.5919165, 51.0798833 ] ] } },
-    { "type": "Feature", "properties": { "river_name": "VENLOOP", "source": "EcoInv" }, "geometry": { "type": "LineString", "coordinates": [ [ 4.6443172, 51.1940245 ], [ 4.644403, 51.1938051 ], [ 4.6439364, 51.1937415 ], [ 4.6438717, 51.1936806 ], [ 4.6439146, 51.1934056 ] ] } },
+    { "type": "Feature", "properties": { "river_name": "WOLFPUTBEEK", "source": "VMM" }, "geometry": { "type": "LineString", "coordinates": [ [ 4.05452, 50.8191468 ], [ 4.0544892, 50.8191765 ], [ 4.0543343, 50.8192157 ], [ 4.0541273, 50.8193985 ], [ 4.0540904, 50.8196061 ], [ 4.0541332, 50.8199122 ] ] } },
+    { "type": "Feature", "properties": { "river_name": "OUDE KALE", "source": "VMM" }, "geometry": { "type": "LineString", "coordinates": [ [ 3.5944035, 51.0797983 ], [ 3.5943438, 51.0797931 ], [ 3.5940242, 51.0797446 ], [ 3.593868, 51.0797255 ], [ 3.5938178, 51.079713 ], [ 3.5937577, 51.0796981 ], [ 3.5935501, 51.0796061 ], [ 3.5933518, 51.0794966 ], [ 3.5932562, 51.0794359 ], [ 3.5932225, 51.0794097 ], [ 3.5931799, 51.0793764 ], [ 3.5931636, 51.0793498 ] ] } },
+    { "type": "Feature", "properties": { "river_name": "VENLOOP", "source": "EcoInv" }, "geometry": { "type": "LineString", "coordinates": [ [ 4.6455959, 51.1934881 ], [ 4.6456818, 51.1932687 ], [ 4.6452152, 51.1932051 ], [ 4.6451504, 51.1931441 ], [ 4.6451933, 51.1928692 ] ] } },
```

Comment:
The coordinates have changed (third digit for longitude, which is in the order of 10-100m?). I wonder why. Probably due to changes in PROJ version? I do not see any R code changes which may have caused this.

Reply:
Sure, it will come from updated PROJ. The preferred transformation pipeline from BD72 / Belgian Lambert 72 to WGS 84 did change some time ago, but I don't have the previous state here to compare with. But we can be confident it is better. I looked up some things out of curiosity.

The step where accuracy loss occurs is the datum shift, i.e. where coordinates in one geographical CRS (BD72) are transformed to coordinates in another geographical CRS (WGS 84). Below is an _extract_ (one step in the operation) from the first operation coming out of `projinfo -s EPSG:31370 -t EPSG:4326 --spatial-test intersects -o WKT2:2019`. You can see that the preferred datumshift is a 7-parameter [Helmert transformation](https://en.wikipedia.org/wiki/Helmert_transformation) with an accuracy of 1 m, which I think is already a high accuracy for transformations to/from WGS 84. One of the sources of inaccuracy when using WGS84 is the fact that it uses a datum ensemble (consecutive datum realizations, hence CRS definitions). Moreover, WGS 84 is a dynamic CRS (Earth-fixed, not plate-fixed), which also requires a specification of the epoch of the coordinates if this source of inaccuracy is to be excluded. See also [this](https://www.youtube.com/playlist?list=PLt0-qTVCvEp1ZwKnf8iup320Cvp9AgXso) nice video.

In terms of accuracy, in Europe we would gain from using the ETRS89 datum, since it is a _static_ geographic CRS (plate-fixed) and because projected CRSs exist (like Belgian Lambert 2008) that are based on it (no accuracy loss). The remark below (end of the STEP object) says that for this Helmert transformation actually the parameters from BD72 to **ETRS89** (`EPSG:4258`) are used (a transformation with an accuracy of 0.2 m).

So all in all, GeoJSON's choice for WGS 84 is fine in an international and general data science perspective, but not so for long-term applications that need high accurcay, such as defining the corner of your permanent quadrat in a long-term study, for example... That is because the same WGS 84 coordinates will not point to the same 'ground' location over the course of years.

```py
    STEP[
        COORDINATEOPERATION["BD72 to WGS 84 (3)",
            VERSION["IGN-Bel 1m"],
            SOURCECRS[
                GEOGCRS["BD72",
                    DATUM["Reseau National Belge 1972",
                        ELLIPSOID["International 1924",6378388,297,
                            LENGTHUNIT["metre",1]]],
                    PRIMEM["Greenwich",0,
                        ANGLEUNIT["degree",0.0174532925199433]],
                    CS[ellipsoidal,2],
                        AXIS["geodetic latitude (Lat)",north,
                            ORDER[1],
                            ANGLEUNIT["degree",0.0174532925199433]],
                        AXIS["geodetic longitude (Lon)",east,
                            ORDER[2],
                            ANGLEUNIT["degree",0.0174532925199433]],
                    ID["EPSG",4313]]],
            TARGETCRS[
                GEOGCRS["WGS 84",
                    ENSEMBLE["World Geodetic System 1984 ensemble",
                        MEMBER["World Geodetic System 1984 (Transit)"],
                        MEMBER["World Geodetic System 1984 (G730)"],
                        MEMBER["World Geodetic System 1984 (G873)"],
                        MEMBER["World Geodetic System 1984 (G1150)"],
                        MEMBER["World Geodetic System 1984 (G1674)"],
                        MEMBER["World Geodetic System 1984 (G1762)"],
                        MEMBER["World Geodetic System 1984 (G2139)"],
                        ELLIPSOID["WGS 84",6378137,298.257223563,
                            LENGTHUNIT["metre",1]],
                        ENSEMBLEACCURACY[2.0]],
                    PRIMEM["Greenwich",0,
                        ANGLEUNIT["degree",0.0174532925199433]],
                    CS[ellipsoidal,2],
                        AXIS["geodetic latitude (Lat)",north,
                            ORDER[1],
                            ANGLEUNIT["degree",0.0174532925199433]],
                        AXIS["geodetic longitude (Lon)",east,
                            ORDER[2],
                            ANGLEUNIT["degree",0.0174532925199433]],
                    ID["EPSG",4326]]],
            METHOD["Coordinate Frame rotation (geog2D domain)",
                ID["EPSG",9607]],
            PARAMETER["X-axis translation",-106.8686,
                LENGTHUNIT["metre",1],
                ID["EPSG",8605]],
            PARAMETER["Y-axis translation",52.2978,
                LENGTHUNIT["metre",1],
                ID["EPSG",8606]],
            PARAMETER["Z-axis translation",-103.7239,
                LENGTHUNIT["metre",1],
                ID["EPSG",8607]],
            PARAMETER["X-axis rotation",-0.3366,
                ANGLEUNIT["arc-second",4.84813681109536E-06],
                ID["EPSG",8608]],
            PARAMETER["Y-axis rotation",0.457,
                ANGLEUNIT["arc-second",4.84813681109536E-06],
                ID["EPSG",8609]],
            PARAMETER["Z-axis rotation",-1.8422,
                ANGLEUNIT["arc-second",4.84813681109536E-06],
                ID["EPSG",8610]],
            PARAMETER["Scale difference",-1.2747,
                SCALEUNIT["parts per million",1E-06],
                ID["EPSG",8611]],
            OPERATIONACCURACY[1.0],
            ID["EPSG",15929],
            REMARK["Parameter values from BD72 to ETRS89 (2) (code 15928) assuming that ETRS89 is equivalent to WGS 84 within the accuracy of the tfm. Scale difference is given by information source as -1.0000012747; given in this record in ppm to assist application usage."]]]
```